modules/activity-tracker-instance/versions.tf

```diff
@@ -22,7 +22,8 @@ terraform {
   required_version = ">=0.13"
   required_providers {
     ibm = {
-      source = "IBM-Cloud/ibm"
+      source  = "IBM-Cloud/ibm"
```

Comment:
bump the `required_version = ">= 1.0.0"`

Reply:
This part is in the comment block, but I can change it so we have a good example.

Reply:
`required_version` is not commented out?

Reply:
oh good catch, this is in one of the submodules, which I might have missed! I'll double check all of them.

Reply:
Fixed the required_version in all of the submodules to match the parent in the latest commit